archive/tar_test.go

```diff
@@ -1156,8 +1158,39 @@ func TestDiffTar(t *testing.T) {
 	}
 }
 
+func TestWhiteoutSourceDateEpoch(t *testing.T) {
+	if v, _ := epoch.SourceDateEpoch(); v != nil {
+		t.Skipf("SOURCE_DATE_EPOCH is already set: %v", v)
+	}
+	sourceDateEpoch, err := time.Parse(time.RFC3339, "2022-01-23T12:34:56Z")
+	require.NoError(t, err)
+	epoch.SetSourceDateEpoch(sourceDateEpoch)
+	t.Cleanup(epoch.UnsetSourceDateEpoch)
+	validators := []tarEntryValidator{
+		composeValidators(whiteoutEntry("f1"), requireModTime(sourceDateEpoch)),
+	}
```

Comment:
I think I'm missing something; why do we need `composeValidators` when `validators` is already a `[]tarEntryValidator`?  Can we do `validators := []tarEntryValidator{whiteoutEntry("f1"), requireModTime(sourceDateEpoch)}` instead?

Reply:
> why do we need `composeValidators` when `validators` is already a `[]tarEntryValidator`?

Because single validator for single file.

>  Can we do `validators := []tarEntryValidator{whiteoutEntry("f1"), requireModTime(sourceDateEpoch)}` instead?

No, that means validating two different files.